### PR TITLE
Minutes 21 07 2022

### PR DESCRIPTION
**Pull request by Dedekind561**
## What does this change?

Syllabus minutes - 21-07-2022

## Rendered Pages

-----
[View rendered docs/contributing/minutes.md](https://github.com/CodeYourFuture/syllabus/blob/minutes_21_07_2022/docs/contributing/minutes.md)

### CODE REVIEW

File: docs/contributing/minutes.md [hunk 1]
```diff
@@ -4,6 +4,66 @@ title: Team Minutes
...
 
+## 2022-07-21
+
+Attendees: Sally, Apoorva, Greg, Mitch
+
+### ❗Actions from last time
+
+- [ ] Sally: send list of some more people for React module team
+- [ ] Ali: chat with Lily about more people for React module team
+- [ ] Ali: coordinate with applicants so far for React module team
```
Comment (alismith-dev) [inline docs/contributing/minutes.md]: I think these can all be marked as done - we have a React module team formed now.

File: docs/contributing/minutes.md [hunk 2]
```diff
@@ -4,6 +4,66 @@ title: Team Minutes
...
+- [ ] Sally: follow up with ZA at some point about why they needed to conduct interviews
+
+### Fundamentals feedback
+
+- Feels like we need a lot of TA support for Fundamentals
+- Add something to the module coordinator notes to think about the number of TAs required for a Fundamentals session
+- TAs not assigned to a room in Scotland - so they all jump around
+- They need experienced people to guide them when they gooff posted
```
Comment (alismith-dev) [inline docs/contributing/minutes.md]:
```suggestion
- They need experienced people to guide them when they go off posted
```

Not sure if there's a missing word at the end too?

File: docs/contributing/minutes.md [hunk 3]
```diff
@@ -4,6 +4,66 @@ title: Team Minutes
...
+- LDN9 is a completely new group of volunteers - so good test of our unspoken assumptions
```
Comment (alismith-dev) [inline docs/contributing/minutes.md:24]: Don't think this needs to go into the notes, but from talking with some LDN8 tech ed volunteers, I think some are planning to help out. I got the impression that they're not aware of explicit timelines around LDN9. And not necessarily volunteering in leadership roles.

File: docs/contributing/minutes.md [hunk 4]
```diff
@@ -4,6 +4,66 @@ title: Team Minutes
...
+- A lot of pushback from trainees, why are we using Blocks - like kids stuff
+- Found the Blockly fundamentals homework overwhelming
+- Greg: The split up of the homework was arbitrary - so we can rearrange it so we can add some more homework to Week 1 if required, for example
+- A section on MVP in Week 2 - not realistic for it to be completed in 10 mins - more like 20 & 20 or 15 & 15
+- Students said stuff "user stories" wasn't important
+- Fundamentals will give students a better though far from complete impression of what being a software engineer involves
+- Come up with a concrete outcome for the coursework - as trainees struggle to follow more abstract instructions
+- Trainees are very wary of people in institutions - experience of being lied to etc
+- Some things like create a discussion in a slack thread aren't happening
+- Smaller spaces with games and retreat spaces good for breaking up large classes with 50+ peple
```
Comment (alismith-dev) [inline docs/contributing/minutes.md]:
```suggestion
- Smaller spaces with games and retreat spaces good for breaking up large classes with 50+ people
```

File: docs/contributing/minutes.md [hunk 5]
```diff
@@ -4,6 +4,66 @@ title: Team Minutes
...
+- See link for more detail -> https://docs.google.com/document/d/1hNolfVFbwb3mYML3OaNw1zStACZbw5np-Obpx6eLtJ4/edit
+- Course structure here -> https://docs.google.com/document/d/1V6HEu_OcJ3MHH-aHzUfANf06VJa1rPcGHcpBwql7QLA/edit#heading=h.dgaqq1am7dqu
+- Invitation has been sent out to trainees
+- Program is designed around x5 10 day sprints
+- Hardly anyone has applied for the course so far - despite there being no consequences for applying
+
+### Teacher training
+
+- Currently running 2 iterations of the teacher training course in West Midlands and North West. West Midlands -> completely remote, North West -> Hybridg.
+- Current format involves 4/5 x 2 hour sworkshops ( with plenty .of attendee participation and engagement). Using Miro to collect ideas - working well thus far.
```
Comment (alismith-dev) [inline docs/contributing/minutes.md]:
```suggestion
- Currently running 2 iterations of the teacher training course in West Midlands and North West. West Midlands -> completely remote, North West -> Hybrid.
- Current format involves 4/5 x 2 hour workshops ( with plenty .of attendee participation and engagement). Using Miro to collect ideas - working well thus far.
```

**Review by alismith-dev — COMMENTED** [review]
Noticed a couple of typos, but thanks for taking the notes!

**Comment by Dedekind561** [comment]
@40thieves Forgot to resolve these conversations before the break - apologies. Just pushed a few updates so we can get these in

**Review by alismith-dev — APPROVED** [review]
LGTM :shipit:

**Comment by 40thieves** [comment]
I think we can just merge this now?

**Pull request merged**